isic_archive/models/image.py

```diff
@@ -91,12 +126,15 @@ def createImage(self, imageDataStream, imageDataSize, originalFileRelpath,
         # TODO: copy license from dataset to image
         return image
 
-    def originalFile(self, image):
-        if 'largeImage' in image:
-            return File().load(image['largeImage']['originalId'], force=True)
-        else:
-            # Fallback if no large image metadata exists, but this isn't accurate on some old images
-            return Image().childFiles(image, limit=1, sort=[('created', SortDir.ASCENDING)])[0]
+    def originalFile(self, image) -> Optional[dict]:
+        original = File().findOne({
+            'itemId': image['_id'],
+            'imageRole': 'original'
+        })
+
+        if original:
```

Comment:
I still don't understand why this is happening. Isn't `original` simply the thing we want to return?